src/components/PeopleFilters.tsx

```diff
@@ -16,6 +79,8 @@ export const PeopleFilters = () => {
             type="search"
             className="input"
             placeholder="Search"
+            value={query}
+            onChange={(e) => handleQueryChange(e)}
```

Comment:
```suggestion
            onChange={handleQueryChange}
```
you should pass a function that accepts the event as an argument - like your `handleQueryChange` function, the additional wrapper is redundant - FYI